docs/sidebar.yaml

```diff
@@ -23,6 +23,10 @@
     - - Building an Analytics Funnel
       - 'https://blog.theodo.com/2018/01/building-google-analytics-funnel-firebase-react-native'
   - '//static.invertase.io/assets/firebase/analytics.svg'
+- - AppCheck
+  - - - Usage
+      - '/app-check/usage'
+  - '//static.invertase.io/assets/social/firebase-logo.png'
```

Comment:
I patterned the sidebar stuff off the rest of them, but it is simply not linking in :sob: